jax/experimental/jax2tf/tests/primitive_harness.py

```diff
@@ -2048,7 +2048,7 @@ def _make_select_and_scatter_add_harness(name,
     padding=padding)
 
 
-for dtype in set(jtu.dtypes.all) - set([np.complex64, np.complex128]):
+for dtype in set(jtu.dtypes.all) - set([np.bool_, np.complex64, np.complex128]):
```

Comment:
Without this change, we get a failure in an abstract eval rule, saying "add does not accept dtype bool". (This change means we run abstract eval rules when compiling single-primitive jaxprs in cases we may not have before.)

Either this test is wrong or the abstract eval rule is wrong. That is, should add accept bool inputs?

Reply:
I believe an internal CI test failed on this, so I'm going to update the abstract eval rule.